### PR TITLE
Brig integration tests

### PR DESCRIPTION
**Pull request by jschaul**
Brig integration/end2end initial test for federation

For easier review, this PR is a small piece from the now-long-running "federation" branch #1319 with the following purpose:

* refactor some brig-integration setup code (remove looking at environment
  variables)
* brig now knows about federator's existence in the config file. This is
  an optional field. (should it be required?)
* introduce a first test that needs two backends. This test is failing
  on this branch (but working to success on #1319 currently), but will
  be disabled-by-default to run both on CI and locally, as long as this PR
  is merged *after* #1341
* prepare some federation chart config. Note there is currently a `federator` and `federatorInternal` config here: the former can be deleted as soon as some PRs involving the actual federator have made their way in, but it's kept in here to keep CI happy for the time being.

### CODE REVIEW

**Review by akshaymankar — APPROVED** [review]

**Review by smatting — APPROVED** [review]

**Comment by jschaul** [comment]
Thanks for the review. Just waiting on CI to make sure it passes before merging.

**Comment by jschaul** [comment]
CI integration test failure (only in spar integration tests) will be fixed with https://github.com/wireapp/wire-server/pull/1343.

**Pull request merged**
